synse/version.go

```diff
@@ -0,0 +1,49 @@
+package synse
+
+// version.go defines version information of the client.
```

Comment:
We shouldn't need this file since this is just a client library, not a standalone binary. If we're not building anything, there will be no build-time information to use here.

Reply:
I am not totally clear on why a client library like this should not need versioning. I was thinking about the client library that we used recently, namely [kubernetes/client-go](https://github.com/kubernetes/client-go), and thought we could have such similar versioning so we would know what version to point to. For example, as in blackbox, we used `v8.0.0.` k8s client.

Reply:
We'll have versioning for sure, but that will happen at a higher level using things like github tags. This pattern is used for things that produce binaries (e.g. like CLI, blackbox, etc) that run a program and provides a way to expose that version information at runtime (note that it only exposes it, it doesn't actually define the version). Since this will just be a library that other Go projects will use (e.g. this won't have its own binary), we can't really use this to expose the version in a meaningful way anyways. 

For this project, using GitHub tags should be enough to designate a version for the library. Then when a different project uses it, we can vendor it to that version, e.g.
```toml
[[constraint]]
  name = "github.com/vapor-ware/synse-client-go"
  version = "1.0.0"
```